assets/src/components/incomingBox.tsx

```diff
@@ -42,6 +45,7 @@ const IncomingBoxVehicle = ({
           status={drawnStatus(vehicleOrGhost)}
         />
       </div>
+      {alertIconStyle ? <IconAlertCircle style={alertIconStyle} /> : null}
```

Comment:
I'm not sure the best words to describe it, but this feels like "logic as a side effect" to me in a way that feels problematic. That is, I think there are two questions here: a) should I display an alert icon?, and b) if so, what style icon should I use? But they are both encoded in `alertIconStyle` in a way that doesn't seem intuitive to me.

Reply:
The idea is that if `alertIconStyle` is undefined, that means "don't draw the icon at all". I can make that clearer by doing a `alertIconStyle === undefined` check and by declaring the type `const alertIconStyle : AlertIcontyle | undefined`.